lib/fastlane/plugin/mango/helper/mango_helper.rb

```diff
@@ -73,10 +73,11 @@ def setup_container
 
       def kvm_disabled?
         begin
-         docker_exec('kvm-ok > kvm-ok.txt')
+          @docker_commander.docker_exec(command: 'kvm-ok > kvm-ok.txt', container_name: container_name)
         rescue StandardError
+          # kvm-ok will always throw regardless of the result. therefore we save the output in the file and ignore the error
```

Comment:
What? There must be a better way that this.

Reply:
there probably is. I think it raises because of `tty` issues.. I spent some time with that and decided to go this way as it took too much time to investigate

Reply:
this is thrown a lot during the plugin work: `ttyname failed: Inappropriate ioctl for device` I can imagine that this is the reason of the error